packages/ariakit-playground/src/playground-code.tsx

```diff
@@ -212,7 +212,7 @@ export type PlaygroundCodeOptions<T extends As = "div"> = Options<T> & {
   maxHeight?: number;
   disclosureProps?: ButtonProps<ElementType>;
   expanded?: boolean;
-  setExpanded?: SetState<boolean>;
+  setExpanded?: (expanded: boolean) => void;
```

Comment:
Changing this type so it doesn't need to be a state updater function similar to the one returned by React's `useState`. That is, people can pass any stable or non-stable function that accepts a boolean value.